mParticle-ROKOMobi/MPKitROKOMobi.m

```diff
@@ -0,0 +1,207 @@
+//
+//  MPKitROKOMobi.m
+//
+//  Copyright 2017 ROKO Labs, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import "MPKitROKOMobi.h"
+
+@interface MPKitROKOMobiProxy: NSObject <MPKitROKOMobiProvider>
+
+@property (nonatomic, strong) ROKOInstaBot *instabot;
+@property (nonatomic, strong) ROKOLinkManager *linkManager;
+
+@end
+
+@implementation MPKitROKOMobiProxy
+
+- (ROKOInstaBot *)getInstaBot {
```

Comment:
we already have the notion of `providerKitInstance` - can developers just use that?

Reply:
I'm unclear on your concern?

The primary purpose of the `MPKitInstabotProxy` is to provide access to the two major components of our sdk functionality (getInstabot, getLinkManager). `getInstabot` and `getLinkManager` are *only* accessible to the developer through `providerKitInstance`.

Reply:
nvm I see now - thanks!